mbofui/app/views/myevents.html

```diff
@@ -0,0 +1,16 @@
+<h4> List of events will show up here</h4>
+
+<!-- <ul class="list-group">
+  <li class="list-group-item" ng-repeat="event in myEvents">
+    <h5>
+      {{event.category}}
+    </h5>
+    <div class="">
+      {{event.message}}
+    </div>
+    <div class="">
+      <small>{{event.startTime | date:'MM/dd h:mma'}} - {{event.endTime | date:'MM/dd h:mma'}}</small>
+    </div>
+  </li>
+</ul>
+ -->
```

Comment:
Yeah. I modeled this based on dummy data and liked the results. Did not want to plumb through history to restore it when the real data becomes available. Lazy!